Иванов Дмитрий

Lurtz1994 opened the pull request.

Lurtz1994 committed:
add solution

ulitcos commented on index.html:
```diff
@@ -1,3 +1,231 @@
+    <title>Бабушкина коллекция</title>
+</head>
+<body>
+    <section>
```
main, header

ulitcos commented on index.html:
```diff
@@ -1,3 +1,231 @@
+                    <li>Дата размещения: <span itemprop="datePublished">8 котобря 2015</span></li>
+                    <li>Жанр: <span itemprop="genre">Русская классика</span>, <span itemprop="genre">
+                        Литература 19 века</span>
+                    </li>
```
это не список. это информация о произведении

ulitcos commented on index.html:
```diff
@@ -1,3 +1,231 @@
+            <h2>Война и мир.</h2>
+            <img src="http://fb2-txt.ru/wp-content/uploads/2015/07/oblogka.jpg" title="Война и мир"
+                 alt="Обложка книги" height="300">
+            <section itemscope itemtype="http://schema.org/book">
```
больше на article похоже, чем на section

ulitcos commented on index.html:
```diff
@@ -1,3 +1,231 @@
+                <ul>
+                    <li>Тип: Книга</li>
+                    <li>Издательство: <span itemprop="publisher">АСТ, Астрель</span></li>
+                    <li>ISBN: <span itemprop="isbn">5-17-006400-4</span>,
```
аббревиатура

ulitcos commented on index.html:
```diff
@@ -1,3 +1,231 @@
+        </article>
+        <hr>
+        <article>
+            <h2>Harry Potter and the Sorcerer's Stone</h2>
```
текст на английском, можно добавить lang

ulitcos commented on index.html:
```diff
@@ -1,3 +1,231 @@
+            <img
+                src="https://upload.wikimedia.org/wikipedia/en/6/6b/Harry_Potter_and_the_Philosopher's_Stone_Book_Cover.jpg"
+                title="Harry Potter and Philosopher's stone" alt="Обложка книги" height="300">
+            <section itemscope itemtype="http://schema.org/book">
```
по ссылке http://schema.org/book - 404

ulitcos commented on index.html:
```diff
@@ -1,3 +1,231 @@
+                        <li>Род деятельности: <span itemprop="jobTitle">прозаик</span></li>
+                        <li>URL: <a href="http://jkrowling.com">jkrowling.com</a></li>
+                        <li>Адрес: <span itemprop="address">50 Bedford Square, London</span></li>
+                        <li>Email: <span itemprop="email">info@jkrowlingpr.com</span></li>
```
контактные данные автора, есть специальные теги

ulitcos commented on index.html:
```diff
@@ -1,3 +1,231 @@
+    <form>
+        <h3>Форма для ввода информации о книге</h3>
+        <ul>
+            <li>Издательство: <input type="text">
```
без name использовать input будет очень тяжело

ulitcos commented on index.html:
```diff
@@ -1,3 +1,231 @@
+            <li>Издательство: <input type="text">
+            <li>ISBN: <input type="number">
+            <li>Год: <input type="number" max="2015"></li>
+            <li>Язык: <input type="text"></li>
```
можно select применить

ulitcos commented on index.html:
```diff
@@ -1,3 +1,231 @@
+            <li>Год: <input type="number" max="2015"></li>
+            <li>Язык: <input type="text"></li>
+            <li>Кол-во страниц: <input type="number"></li>
+            <li>Формат: <input type="text"></li>
```
select

ulitcos commented on index.html:
```diff
@@ -1,3 +1,231 @@
+            <li>Формат: <input type="text"></li>
+            <li>Переплет: <input type="text"></li>
+            <li>Дата размещения: <input type="date"></li>
+            <li>Жанр: <input typr="text"></li>
```
можно сделать лучше http://htmlbook.ru/html/select/multiple

ulitcos commented on index.html:
```diff
@@ -1,3 +1,231 @@
+            <li>Жанр: <input typr="text"></li>
+            <li>URL: <input type="url"></li>
+            <li>Описание: <textarea rows="3" cols="50" placeholder="Введите описание"></textarea></li>
+        </ul>
```
это не список

Lurtz1994 committed:
fixed

Lurtz1994 commented:
@ulitcos

ulitcos commented on index.html:
```diff
@@ -1,3 +1,224 @@
+                <h2>Война и мир.</h2>
+                <img src="http://fb2-txt.ru/wp-content/uploads/2015/07/oblogka.jpg" title="Война и мир"
+                     alt="Обложка книги" height="300"><br>
+                <section itemscope itemtype="http://schema.org/Book">
```
заголовок и картинка тоже должны быть внутри `itemscope itemtype="http://schema.org/Book"`

перенеси в article

ulitcos commented on index.html:
```diff
@@ -1,3 +1,224 @@
+                    Переплет: Твердый переплет<br>
+                    Дата размещения: <span itemprop="datePublished">8 котобря 2015</span><br>
+                    Жанр: <span itemprop="genre">Русская классика</span>, <span itemprop="genre">
+                        Литература 19 века</span><br>
```
Жанр: <span>...</span>
           <span>...</span>

ulitcos replied:
второй span на новую строку

ulitcos commented on index.html:
```diff
@@ -1,3 +1,224 @@
+                    URL:
+                        <a href="http://babulenkin-servant.com/books/war_and_peace.epub">
+                            http://babulenkin-servant.com/books/war_and_peace.epub
+                        </a><br>
```
itemprop="url"

ulitcos commented on index.html:
```diff
@@ -1,3 +1,224 @@
+                        как огромная мозаика, сложен из тысяч картин - батальных, бальных, любовных, бытовых,
+                        политических, философских, портретных, пейзажных, психологических...</span><br>
+                    Фрагмент аудиокниги:
+                        <audio controls src="http://audiohrestomatiya.ru/get/ba128d5ba6a11708e6a3896e0e909097/">
```
itemprop="audio"

ulitcos replied:
по ссылке 404

ulitcos commented on index.html:
```diff
@@ -1,3 +1,224 @@
+                    Фрагмент аудиокниги:
+                        <audio controls src="http://audiohrestomatiya.ru/get/ba128d5ba6a11708e6a3896e0e909097/">
+                        </audio><br>
+                    <section itemscope itemtype="http://schema.org/Person">
```
itemprop="author"

ulitcos commented on index.html:
```diff
@@ -1,3 +1,224 @@
+                        <audio controls src="http://audiohrestomatiya.ru/get/ba128d5ba6a11708e6a3896e0e909097/">
+                        </audio><br>
+                    <section itemscope itemtype="http://schema.org/Person">
+                        <img src="http://dic.academic.ru/pictures/es/284260.jpg" alt="Портрет Л.Н.Толстого"
```
itemprop="image"

ulitcos commented on index.html:
```diff
@@ -1,3 +1,224 @@
+                            <span itemprop="givenName">Лев</span>
+                            <span itemprop="additionalName">Николаевич</span>
+                            <span itemprop="familyName">Толстой</span></span><br>
+                        Дата рождения: <span itemprop="birthDate">16 августа 1828</span><br>
```
для для дат есть специальный тег

ulitcos commented on index.html:
```diff
@@ -1,3 +1,224 @@
+                        Род деятельности: <span itemprop="jobTitle">прозаик</span><br>
+                        URL: <a href="http://jkrowling.com">jkrowling.com</a><br>
+                        Адрес: <span itemprop="address">50 Bedford Square, London</span><br>
+                        Email: <span itemprop="email">info@jkrowlingpr.com</span><br>
```
для контактных данных есть специальный тег

ulitcos commented on index.html:
```diff
@@ -1,3 +1,224 @@
+                    Страна: <span itemprop="country">США</span><br>
+                    Слоган: <span itemprop="headline">«Ничто на Земле не сможет разлучить их»</span><br>
+                    Режиссер: <span itemprop="director">Джеймс Кэмерон</span><br>
+                    Сценарий: <span itemprop="">Джеймс Кэмерон</span><br>
```
author

ulitcos commented on index.html:
```diff
@@ -1,3 +1,224 @@
+                    Монтаж: Конрад Бафф IV,Джеймс Кэмерон,Ричард А. Харрис<br>
+                    Жанр: <span itemprop="genre">драма</span>,
+                        <span itemprop="genre">мелодрама</span><br>
+                    Дата размещения: <span itemprop="">1 котобря 2015</span><br>
```
datePublished

ulitcos commented on index.html:
```diff
@@ -1,3 +1,224 @@
+                    Композитор: <span itemprop="musicBy">Джеймс Хорнер</span><br>
+                    Художник: <span itemprop="">Питер Ламонт</span>,
+                        <span itemprop="">Мартин Лэйн</span>,
+                        <span itemprop="">Чарльз Дуайт Ли</span><br>
```
лучше убрать, если путой

ulitcos commented on index.html:
```diff
@@ -1,3 +1,224 @@
+            <abbr title=" International Standard Book Number">
+                ISBN</abbr>: <input name="isbn" type="number"><br>
+            Год: <input name="year" type="number" max="2015"><br>
+            Язык: <select name="cover">
```
больше на lang похоже, чем на cover

ulitcos commented on index.html:
```diff
@@ -1,3 +1,224 @@
+            </select><br>
+            Кол-во страниц: <input name="numberOfPages" type="number"><br>
+            Формат: <input name="bookFormat" type="text"><br>
+            Переплет: <select name="cover">
```
второй раз cover

ulitcos commented on index.html:
```diff
@@ -1,3 +1,224 @@
+                    <option>Детское</option>
+                    <option>Фантастика</option>
+                    <option>Фентези</option>
+                </select><br>
```
+

ulitcos commented:
@Lurtz1994 , пользуйся https://webmaster.yandex.ru/microtest.xml  для проверки микроразметки

Lurtz1994 committed:
Исправил все

https://webmaster.yandex.ru/microtest.xml ругается только на даты, что с ними не так, я не понял. Пробовал по разному, все равно ругается.

Lurtz1994 commented:
@ulitcos поисправлял

Lurtz1994 committed:
Update index.html

Lurtz1994 commented:
@ulitcos В некоторых местах исправилось с тегом <time>, в некоторых нет.

ulitcos commented on index.html:
```diff
@@ -1,3 +1,228 @@
+    <main>
+        <section>
+            <article  itemscope itemtype="http://schema.org/Book">
+                <cite><h2 itemprop="name">Война и мир.</h2></cite>
```
cite тут лишний

ulitcos commented on index.html:
```diff
@@ -1,3 +1,228 @@
+                            <span itemprop="additionalName">Николаевич</span>
+                            <span itemprop="familyName">Толстой</span><br>
+                        Дата рождения: <time datetime="1828-08-16" itemprop="birthDate">16 августа 1828</time><br>
+                        Дата смерти: <time datetime="1910-11-7" itemprop="deathDate">7-11-1910</time><br>
```
datetime="1910-11-07"

ulitcos replied:
внутри time можно написать "7 ноября"

ulitcos commented on index.html:
```diff
@@ -1,3 +1,228 @@
+                        Род деятельности: <span itemprop="jobTitle">прозаик</span><br>
+                        URL: <a itemprop="url" href="http://jkrowling.com">jkrowling.com</a><br>
+                        Адрес: <span itemprop="address">50 Bedford Square, London</span><br>
+                        Email: <span itemprop=contactPoint" itemprop="email">info@jkrowlingpr.com</span><br>
```
для контактов используем [address](http://www.w3.org/html/wg/drafts/html/master/semantics.html#the-address-element)

ulitcos commented on index.html:
```diff
@@ -1,3 +1,228 @@
+                    Страниц: <span itemprop="numberOfPages">309</span> стр.<br>
+                    Формат: <span itemprop="bookFormat">pdf</span><br>
+                    Переплет: Мягкий переплет<br>
+                    Дата размещения: <time datetime="2015-10-4" itemprop="datePublished">4-10-2015</time><br>
```
datetime="2015-10-4" забыл "0" перед четверкой. валидатор ругается

ulitcos commented on index.html:
```diff
@@ -1,3 +1,228 @@
+                           controls>
+                        <source
+                            src="http://cache-ekt03.cdn.yandex.net/kp.cdn.yandex.net/2213/kinopoisk.ru-Titanic-268370.mp4"
+                            type="video/mp4" >
```
`itemprop="trailer"`  и `itemprop="video"` надо добавлять к `source`, а не к `video`

Lurtz1994 committed:
Update index.html

ulitcos commented:
Принял   :+1:
